CHANGELOG-developer.asciidoc

```diff
@@ -12,6 +12,21 @@ other Beats should be migrated.
 
 Note: This changelog was only started after the 6.3 release.
 
+=== Beats version 7.7.0
+https://github.com/elastic/beats/compare/v7.6.2..v7.7.0[Check the HEAD diff]
```

Comment:
Hm, this should not be empty. @ycombinator can you check if the output refactorings are mentioned in CHANGELOG-developer.next.asciidoc?

Reply:
Yes, I just checked https://github.com/elastic/beats/blob/master/CHANGELOG-developer.next.asciidoc and the output refactoring are mentioned there:

```
- The `libbeat/outputs/transport` package has been moved to `libbeat/common/transport`. {pull}16734[16734]
- The `libbeat/outputs/tls.go` file has been removed. All exported symbols in that file (`libbeat/outputs.*`) are now available as `libbeat/common/tlscommon.*`. {pull}16734[16734]
- Extract Elasticsearch client logic from `outputs/elasticsearch` package into new `esclientleg` package. {pull}16150[16150]
```

Not sure why they haven't shown up here. Same goes for the move to Go modules as well, right?